src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -625,6 +641,54 @@ TritonInferResult::RawData(
   return Error::Success;
 }
 
+struct triton::client::HttpSslOptions
+ParseHttpSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&
+        ssl_options)
+{
+  struct triton::client::HttpSslOptions http_ssl_options;
+
+  http_ssl_options.verify_peer = ssl_options.ssl_https_verify_peer;
+  http_ssl_options.verify_host = ssl_options.ssl_https_verify_host;
+  http_ssl_options.ca_info = ssl_options.ssl_https_ca_certificates_file;
+  if (ssl_options.ssl_https_client_certificate_type == "PEM") {
+    http_ssl_options.cert_type =
+        triton::client::HttpSslOptions::CERTTYPE::CERT_PEM;
+  } else if (ssl_options.ssl_https_client_certificate_type == "DER") {
+    http_ssl_options.cert_type =
+        triton::client::HttpSslOptions::CERTTYPE::CERT_DER;
+  }
+  http_ssl_options.cert = ssl_options.ssl_https_client_certificate_file;
+  if (ssl_options.ssl_https_private_key_type == "PEM") {
+    http_ssl_options.key_type =
+        triton::client::HttpSslOptions::KEYTYPE::KEY_PEM;
+  } else if (ssl_options.ssl_https_private_key_type == "DER") {
+    http_ssl_options.key_type =
+        triton::client::HttpSslOptions::KEYTYPE::KEY_DER;
+  }
+  http_ssl_options.key = ssl_options.ssl_https_private_key_file;
+
+  return http_ssl_options;
+}
+
+std::pair<bool, struct triton::client::SslOptions>
+ParseGrpcSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&
+        ssl_options)
```

Comment:
remove `struct`

Reply:
good catch--removed here and everywhere else applicable 👍